### PR TITLE
IN16B minor changes

### PR DESCRIPTION
**Pull request by gvardany**
**Description of work.**

Adds an option to choose the inelastic peak channels manually.
Marks monitor normalised QENS and BATS reduced data as distributions.

**To test:**
Changes are minimal and captured by the tests. So code review and `./systemtest -R ILLIndirect` should suffice.

Refs #27148 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v4.3.0/indirect_geometry.rst [hunk 1]
```diff
@@ -49,5 +49,7 @@ BugFixes
 - The Abins file parser no longer fails to read data from non-periodic vibration calculations performed with CRYSTAL17.
 - :ref:`ApplyPaalmanPingsCorrection <algm-ApplyPaalmanPingsCorrection>` will now run also for fixed window scan reduced data and will not crash on workspace groups.
 - Fixed a bug crashing the ``Indirect ILL Data Reduction GUI`` when Run is clicked.
+- Indirect ILL reductions for BATS and Doppler QENS modes will now flag the outputs as distributions if the data is normalised by the monitor.
```
Comment (Harrietbrown) [inline docs/source/release/v4.3.0/indirect_geometry.rst]: Could you also write a test for this change?

**Review by Harrietbrown — DISMISSED** [review]

**Comment by Harrietbrown** [comment]
I'm un-assigning myself from this because I won't be able to approve this before the code freeze at the end of the day.

File: docs/source/release/v4.3.0/indirect_geometry.rst (hunk 1 above)
Comment (gvardany) [reply docs/source/release/v4.3.0/indirect_geometry.rst]: This is captured by the system test, that's why the references are updated.

Comment (NickDraper) [reply docs/source/release/v4.3.0/indirect_geometry.rst]: OK

**Review by NickDraper — APPROVED** [review]
All looks good to go

**Pull request merged**